src/calendar/types.ts

```diff
@@ -0,0 +1,33 @@
+import { Dayjs } from "dayjs";
+
+export type View = "Day" | "Month" | "Year";
+
+export type VariantDay = "default" | "nextMonth" | "today";
+
+export type VariantMonth =
+    | "default"
+    | "currentMonth"
+    | "nextMonth"
+    | "selectedMonth";
+
+export type VariantYear =
+    | "default"
+    | "currentYear"
+    | "nextDecaded"
```

Comment:
I think you mean

```suggestion
    | "nextDecade"
```
can update all other instances of `decaded` too